bullet_train/docs/webhooks/incoming.md

```diff
@@ -5,3 +5,22 @@ Bullet Train makes it trivial to scaffold new endpoints where external systems c
 ```
 rails generate super_scaffold:incoming_webhooks
 ```
+
```

Comment:
We should really flesh out the docs on the upper part of this page. This single paragraph with not even a full example is very much "figure it out for yourself" level of docs. 👎 I can work on a PR for that.